### PR TITLE
add batched logContent endpoint

### PR DESCRIPTION
**Pull request by jnelson15**

### CODE REVIEW

File: src/main/java/com/palantir/weblogger/WebLoggerResource.java [hunk 1]
```diff
@@ -55,7 +56,15 @@ public void logContent(@PathParam("eventName") String eventName, String jsonStri
         jsonLog = addTimestamp(jsonLog);
 
         analyticsLogger.info(jsonLog.toString());
+    }
 
+    @POST
+    @Path("batch")
+    @Consumes(MediaType.APPLICATION_JSON)
+    public void logContent(Map<String, String> events) throws ParseException {
```
Comment (jmcampanini) [inline src/main/java/com/palantir/weblogger/WebLoggerResource.java]: curious why this method is called `logContent`? why not call it something like `logBatch`? or something more related to the path associated.

**Comment by zoubeiri** [comment]
Could we also add a quick line in the README for this so other people can take advantage of it? 
Looking forward to getting this in!

**Comment by jnelson15** [comment]
Yeah no problem

**Comment by zoubeiri** [comment]
Just released v1.3.0 with this included :)

**Comment by jnelson15** [comment]
Thanks!

**Pull request merged**